src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -302,7 +302,19 @@ public static function create($params = [])
 
         MyRadio_Show::getInstance($params['show_id'])->addSeason($season_id);
 
-        return self::getInstance($season_id);
+        $newSeason = self::getInstance($season_id);
+
+        // COVID Studio Usage - Summer 2021
+        if (isset($params['studio-request']) && $params['studio-request']) {
+            MyRadioEmail::sendEmailToList(
+                MyRadio_List::getByName("programming"),
+                "Studio Usage Request",
+                "Season " . $season_id . " of " . $newSeason->getMeta("title")
```

Comment:
This will be something like `Season 85939 of Some Show`, rather than `Season 3 of Some Show`. Is this intented?

Reply:
yes